artichoke-core/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "artichoke-core"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
This version bump comes from:

- Addition of `Intern::SYMBOL_RANGE_START` associated constant.
- Addition of `Intern::symbol_count` associated method.